github/src/main/kotlin/gropius/sync/github/Incoming.kt

```diff
@@ -104,21 +99,43 @@ class Incoming(
 
     /**
      * Save a single timeline event into the database
-     * @param issue the issue the timeline belongs to
+     * @param issueInfo the issue the timeline belongs to
      * @param event a single timeline item
+     * @param imsProjectConfig Config of the active project
      * @return The time of the event or null for error
      */
     suspend fun handleTimelineEvent(
-        imsProjectConfig: IMSProjectConfig, issue: IssueInfo, event: TimelineItemData
+        imsProjectConfig: IMSProjectConfig, issueInfo: IssueInfo, event: TimelineItemData
     ): OffsetDateTime? {
-        val dbEntry = timelineEventInfoRepository.findByGithubId(event.asNode()!!.id)
-        return if ((dbEntry != null) && !((event.asIssueComment()?.lastEditedAt != null) && (event.asIssueComment()!!.lastEditedAt!! > dbEntry.lastModifiedAt))) {
+        val dbEntry = timelineEventInfoRepository.findByUrlAndGithubId(imsProjectConfig.url, event.asNode()!!.id)
+        return if (event.asIssueComment() != null) {
```

Comment:
too long, please split, especially as there seem to be distinct cases with different behavior

Reply:
705ec38fdc40f8ca96b5c7b883c1f841eb66a176